solana/Makefile

```diff
@@ -12,9 +12,15 @@ target/idl/example_native_token_transfers.json:
 _anchor-build:
 	@anchor build --arch sbf
 
-anchor-test: node_modules build target/idl/example_native_token_transfers.json
+anchor-test: node_modules build target/idl/example_native_token_transfers.json sdk
```

Comment:
running `make anchor-test` is not efficient, too much gets built

I think i can put some is-built flags in to speed it up but happy to take suggestions